testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksCreateTableAsSelectCompatibility.java

```diff
@@ -248,4 +249,43 @@ public void testReplaceTableWithSchemaChangeOnCheckpoint()
             dropDeltaTableWithRetry(tableName);
         }
     }
+
+    @Test(groups = {DELTA_LAKE_DATABRICKS, PROFILE_SPECIFIC_TESTS})
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testCreateTableAsSelectWithAllPartitionColumns()
```

Comment:
Can we use other tests in this case? This test will be run only when run with secrets, maybe it would be good to have it run for all PRs?

Reply:
Added tests to `TestDeltaLakeConnectorTest`. Please note that this class is dedicated to Databricks, not OSS Delta Lake for now.